### ISSUE TITLE
Support for iRobot Roomba e5

### ISSUE THREAD

**Issue by Mikulas** [denysdovhan/vacuum-card#12]
Roomba e5 is fairly dumb and has no exposed sensors other than battery and bin status.

```
vacuum.pani_hudsonova

battery_level: 100
battery_icon: mdi:battery-charging-100
software_version: 3.4.59+92
status: Charging
bin_present: true
bin_full: false
friendly_name: Paní Hudsonová
supported_features: 13276
```

The card does not render as it fails on getting those attributes. https://github.com/denysdovhan/vacuum-card/blob/master/src/vacuum-card.js#L116-L152

Ideally all the attributes would be rendered conditionally, but I understand that would be a lot of error. In the mean time I will fork the repo and trim everything not supported on e5.

**Comment by denysdovhan** [comment]
We can conditionally render stats. If we don't have a specific value, just omit to render it. Can you send a PR?

**Comment by vanstinator** [comment]
I have the i7 and will be looking at adding support tonight. I'd be happy to take a look at this while adding support.

**Comment by vanstinator** [comment]
@denysdovhan where did you source the art for the vacuum? I think it might be interesting to show different art that matches the configured vacuum type.

**Comment by denysdovhan** [comment]
@vanstinator from the original cards: https://github.com/denysdovhan/vacuum-card#inspiration

**Comment by denysdovhan** [comment]
Should be resolved by #29. It will be released as 1.0.0.
